frontend/src/app/forms/steps/key-competencies/key-competencies.component.scss

```diff
@@ -0,0 +1,13 @@
+.list-item-label {
+  font-weight: 500;
+  letter-spacing: 1px;
+  font-size: 1.2rem;
+  white-space: normal;
+}
```

Comment:
I think the font-weight is here a bit too strong, 400 would be enough for me.

Reply:
I wanted to make it like in the mockups. So if you are ok with it, I would leave it as it is for now :) The items are pretty important, so I guess it is good to have them stick out a bit